Makefile

```diff
@@ -7,24 +7,35 @@ all: build
 .ONESHELL:
 install:
 	if hash apt-get 2>/dev/null; then
-	  sudo apt-get update -qq >/dev/null && sudo apt-get install -qq apt-utils imagemagick python3-pip python3-setuptools gcc
+	  sudo apt-get update -qq >/dev/null && sudo apt-get install -qq apt-utils imagemagick python3-pip python3-setuptools gcc git-lfs
```

Comment:
Makefiles which modify the system configuration usually do not have the `sudo` calls in them. Instead you are supposed to start `make` with sudo (or your system's equivalent) yourself to avoid surprising changes to user's systems. For `make install` this is usually the case. I'd prefer it if you could remove the `sudo` calls from the Makefile to avoid people accidentally modifying their system.

Reply:
Also: Couldn't we also install lektor from the package repositories instead of installing it globally with `pip`? I have the feeling that this might have unexpected behaviour for people actually using Python on their systems.

Reply:
Besser nicht.

Wenn wir die ``package repositories`` verwenden haben wir mal wieder potentiell komplett unterschiedliche lektor versionen oder im zweifelsfall einfach gar keine weil nicht im offiziellen repo.

Reply:
> Makefiles which modify the system configuration usually do not have the `sudo` calls in them. Instead you are supposed to start `make` with sudo (or your system's equivalent) yourself to avoid surprising changes to user's systems. For `make install` this is usually the case. I'd prefer it if you could remove the `sudo` calls from the Makefile to avoid people accidentally modifying their system.

Das könnte man machen...
ABER die sudo calls sind schon vor diesem commit drinen und gehören doch eher in einen neuen Pull-Request! Daher nein.

Reply:
Naja, du änderst ja sowieso schon einige Sachen hier in der Datei, damit das mit der neuen CI gut funktioniert, da kann man diese Änderung auch gleich mitnehmen, das ist denke ich themenverwandt.